include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -157,6 +169,33 @@ class LAPACKFullMatrix : public TransposeTable<number>
    */
   void reinit (const size_type size);
 
+  /**
+   * Same as above but will preserve the values of matrix upon resizing.
+   * The original values
+   * of the matrix are kept on increasing the size
+   * \f[
+   * \mathbf A \rightarrow
+   * \left(
+   * \begin{array}{cc}
+   * \mathbf A & \mathbf 0 \\
+   * \mathbf 0 & \mathbf 0
+   * \end{array}
+   * \right)
+   * \f]
+   * Whereas if the new size is smaller, the matrix will contain the upper left block
+   * of the original one
+   * \f[
+   * \mathbf A_{11} \leftarrow
+   * \left(
+   * \begin{array}{cc}
+   * \mathbf A_{11} & \mathbf A_{12} \\
+   * \mathbf A_{21} & \mathbf A_{22}
+   * \end{array}
+   * \right)
+   * \f]
```

Comment:
Is there any aesthetic reason to make this second case a `leftarrow`? I would have found putting `A_{11}` on the right with a `rightarrow` more intuitive...

Reply:
will update.